games-roguelike/FTL-gog/FTL-gog-1.6.3.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop unpacker
+
+DESCRIPTION="Top-down roguelike space ship simulator"
+HOMEPAGE="https://www.gog.com/game/faster_than_light"
+SRC_URI="ftl_advanced_edition_en_1_6_3_17917.sh"
+RESTRICT="bindist fetch mirror strip test"
+
+LICENSE="GOG-EULA"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE=""
+
+DEPEND="app-arch/unzip"
+RDEPEND="media-libs/freetype
+	media-libs/libsdl[X,sound,opengl,video]
+	media-libs/libpng
+	sys-libs/zlib
+	virtual/opengl"
+
+S="${WORKDIR}/data/noarch"
+
+pkg_nofetch() {
+	elog "Please buy and download ${SRC_URI} from"
+	elog "https://www.gog.com/game/faster_than_light"
+	elog "and copy it to ${DISTDIR}"
```

Comment:
DISTDIR is forbidden in `pkg_nofetch`.

Reply:
What is the preferred practice in this situation then? Should the same suggestion remain but simply without a reference to what is normally `/usr/portage/distfiles` ? This elog section was based on a set precedent in another ebuild.

Reply:
Just say 'your distfiles directory' or something like that. And yes, ebuilds do that a lot and modern Portage versions output completely useless directories which is not helpful.

Reply:
I had simply dropped the third `elog` line from the ebuild file. I can put the substitute text back in though. If we need another revision I'll put this in with it.